mica/stats/update_acq_stats.py

```diff
@@ -622,7 +622,7 @@ def calc_stats(obsid):
         else:
             raise ValueError(
                 "Problem looking up starcheck for {}".format(obsid)
-            ) from err
+            ) from None
```

Comment:
this is the one case I said above that could be `from err`.

The reason is that if you raise from None, you discard the first exception, which is the one that has the relevant information.

The rule of thumb is: does the first exception add relevant information? If yes, you raise `from err`, if not, raise from None.

Reply:
Sorry, I got lost in above, belows.  I see your point on this one, though I think in general we are not necessarily agreeing on what is useful information.  But that's fine for this ruff PR.